botocore/httpchecksum.py

```diff
@@ -522,8 +522,12 @@ def _handle_bytes_response(http_response, response, algorithm):
     _CRT_CHECKSUM_CLS = {
         "crc32": CrtCrc32Checksum,
         "crc32c": CrtCrc32cChecksum,
-        "crc64nvme": CrtCrc64NvmeChecksum,
     }
+
+    if has_minimum_crt_version((0, 23, 0)):
+        # CRC64NVME support wasn't officially added until 0.23.x
```

Comment:
Is this right? I installed `awscrt==0.22.4` and was able to import `crc64nvme` from the `awscrt.checksums` module.
```py
>>> import awscrt
>>> awscrt.__version__
'0.22.4'
>>> from awscrt.checksums import crc64nvme
>>> 
```

The version we used when updating flexible checksum defaults was `0.23.4`, but it seems like `crc64nvme` has existed for a while now.

Reply:
That's why I used the wording "officially". There were some testing versions earlier but I think it's best to put the floor where we'd stated we now had CRC64NVME support. That prevents us using older copies that may have bugs.

That said, it looks like there were further changes after 0.23.0 and the actual official release is what we use (0.23.4). Since we're not following SemVer here, I'll update the version number to be exact.

Reply:
Should be updated.